src/harness/fourslashImpl.ts

```diff
@@ -942,7 +942,7 @@ namespace FourSlash {
             expected = typeof expected === "string" ? { name: expected } : expected;
 
             if (actual.insertText !== expected.insertText) {
-                this.raiseError(`Expected completion insert text to be ${expected.insertText}, got ${actual.insertText}`);
+                this.raiseError(`Completion insert text did not match: ${showTextDiff(expected.insertText || "", actual.insertText || "")}`);

```

Comment:
that's awesome, I've been needing this for a while

Reply:
I usually try to fix things that annoy me about fourslash, it has definitely paid off even if I were the only person using it 😅